images/flink/docker/Dockerfile

```diff
@@ -29,6 +31,23 @@ RUN echo "Downloading ${GCS_CONNECTOR_URI}" && \
 RUN echo "Downloading ${FLINK_HADOOP_JAR_URI}" && \
   wget -q -O /opt/flink/lib/${FLINK_HADOOP_JAR_NAME} ${FLINK_HADOOP_JAR_URI}
 
+# Install Python and pyflink .
+RUN if [ "${PYTHON_VERSION}" != "" ]; then \
+  apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev libffi-dev && \
+  wget https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tgz && \
+  tar -xvf Python-${PYTHON_VERSION}.tgz && \
+  cd Python-${PYTHON_VERSION} && \
+  ./configure --without-tests --enable-shared && \
+  make -j6 && \
+  make install && \
+  ldconfig /usr/local/lib && \
+  cd .. && rm -f Python-${PYTHON_VERSION}.tgz && rm -rf Python-${PYTHON_VERSION} && \
+  ln -s /usr/local/bin/python3 /usr/local/bin/python && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/* && \
+  pip3 install apache-flink==${FLINK_VERSION}; \
+  fi
+
```

Comment:
I think it would be better if we move this into a separate img ... ie. having a JVM and Python

Reply:
I have added Dockerfile.python